extensions/rates/POSTINSTALL.md

```diff
@@ -0,0 +1,61 @@
+### See it in action
+
+You can test out this extension right away!
+
+1.  Go to your [Cloud Firestore dashboard](https://console.firebase.google.com/project/${param:PROJECT_ID}/firestore/data) in the Firebase console.
+
+1.  If it doesn't already exist, create the collection you specified during installation: `${param:COLLECTION_PATH}`.
+
+1.  You can now use the [Firebase Admin SDK][admin_sdk] to add a document:
+
+```js
+admin
+  .firestore()
+  .collection('${param:COLLECTION_PATH}')
+  .add({
+    shipment: {
+      validateAddress: 'no_validation',
```

Comment:
You're missing some important required fields: `carrierId`, `serviceCode`, and `shipDate`.

Reply:
Ah, actually – I thought this was the `purchaseLabel` extension. So nevermind.